editors/code/src/main.ts

```diff
@@ -287,7 +287,7 @@ async function getServer(config: Config, state: PersistentState): Promise<string
     if (config.package.releaseTag === null) return "rust-analyzer";
 
     let platform: string | undefined;
-    if (process.arch === "x64" || process.arch === "ia32") {
+    if (process.arch === "x64" || process.arch === "ia32" || process.arch === "arm64") {
```

Comment:
Hmm, I see an issue with this - It would allow it to download even on arm64 Windows and Linux (which both exist), and that would require separate binaries, as they don't have Rosetta 2.